<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZStack</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(158, 158, 158);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="Stage_Group">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Stage Controls</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="1" column="0">
           <widget class="QPushButton" name="Set_New_Coord_origin_button">
            <property name="text">
             <string>Set New Zero</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="Go_To_Stage_Origin_button">
            <property name="text">
             <string>Go To  Zero</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="origin_stored_checkbox">
            <property name="text">
             <string>Origin Coords Stored</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="stage_mtoward_button">
            <property name="text">
             <string>Toward sample</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="zstep_box">
            <property name="maximum">
             <number>99999999</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stage_maway_button">
            <property name="text">
             <string>Away sample</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="2">
           <widget class="QPushButton" name="stage_mup_button">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Up</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="stage_mdown_button">
            <property name="text">
             <string>Down</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="stage_mleft_button">
            <property name="text">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="xystep_Box">
            <property name="maximum">
             <number>9999999</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="stage_mmid_button">
            <property name="text">
             <string>Mid</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="stage_mright_button">
            <property name="text">
             <string>Right</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="get_stage_position_button">
            <property name="text">
             <string>Position</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QPushButton" name="disconnect_stage_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="lr_pos_display">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="stage_feature_button">
            <property name="text">
             <string>Cryostat view</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLCDNumber" name="ud_pos_display">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLCDNumber" name="z_pos_display">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Left/Right</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Up/Down</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_12">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>z</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1">
           <widget class="QSpinBox" name="zstep_box_2">
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Z step size</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="take_z_stack_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Run z-stack</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Num steps</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="num_steps_box">
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
